docs/website/docs/plus/features/data-access.md

```diff
@@ -0,0 +1,168 @@
+---
+title: "Secure data access and sharing"
+description: Provide secure data access to your organization
+keywords: ["data access", "security", "contracts", "data sharing"]
+---
+
+import Link from '../../_plus_admonition.md';
+
+<Link/>
+
+# Secure data access and sharing
+
+dlt+ makes it easy for end-users like data scientists or analysts to access high quality production data in a secure and Python-friendly way. A [dlt+ Project](../core-concepts/project.md) exposes a standard Python API which connects to the production data using an "access" [profile](../core-concepts/profiles.md). This profile can be configured to specify how users are allowed to interact with the data, e.g. by applying restrictions on datasets that are not allowed to be modified. 
```

Comment:
please remove this chapter until we have python package layout implemented OR at least make note how users can get this feature to work. because we do not support it in CLI atm.

Reply:
Agreed.

Reply:
If you want to keep it, we need to reference an example repo with a dlt project as a package so the user has some path to success if they really want to.

Reply:
Ok, another note: We could explain how to get the catalog from the project context in python code without the package structure, no?